source/macos/AdaptiveCards/AdaptiveCards/Views/ACRCollectionView.swift

```diff
@@ -59,32 +66,49 @@ class ACRCollectionView: NSScrollView {
         autohidesScrollers = true
     }
     
+    public func containImageViewCell(with id: String) -> NSView? {
```

Comment:
Nit: 
```suggestion
    public func imageCell(with id: String) -> NSView? {
```
If it starts with contains, we usually expect return value to be bool. If we are returning the view, change func name